esp/services/esdl_svc_engine/esdl_store.cpp

```diff
@@ -729,77 +729,99 @@ class CEsdlSDSStore : implements IEsdlStore, public CInterface
         return true;
     }
 
-    virtual int bindService(const char* bindingName,
+    void validatePort(const char *espPort)
```

Comment:
I'm surprised we don't already have a utility like this, perhaps move this up to a more common place?

Reply:
Agreed, except I think it needs thought about how it might be used elsewhere and how it should treat errors, exceptions, etc. and I don't really have time now.  I've really just move the code and made it a function.  Later we could refactor a bit and common it up.  Any thoughts on where it would go?

Reply:
Any reason not to move it up to esp/services/common?
Not critical by any means, but without the 'Esp' prefixes why not move it up to the system/jlib level?